README.md

```diff
@@ -32,42 +32,63 @@ NOTE: underlying third party SDK needs to be replaced as out of date and has a s
 
 // Setup - get the key value (<SECRET>) separately from a vault or
 // environment variable.
-Seneca()
+const seneca = Seneca({ legacy: false })
+  .test() // sets up the Seneca instance for testing: easier to read layout
+  .use('promisify')
+  .use('entity')
   // Get API keys using the seneca-env plugin
   .use('env', {
     var: {
-      $STYTCH_APIKEY: String,
-      $STYTCH_USERTOKEN: String,
+      $STYTCH_PROJECT_ID: String,
```

Comment:
let's keep the format: PROVIDER_VALNAME - only one underscore
thus: STYTCH_PROJECT_ID